src/components/DatePickerRange.react.js

```diff
@@ -263,6 +266,11 @@ DatePickerRange.propTypes = {
     persistence_type: PropTypes.oneOf(['local', 'session', 'memory']),
 };
 
+DatePickerRange.persistenceTransforms = {
+    end_date,
```

Comment:
Assign the value to the prop instead of assigning to a value that matches the prop for shortcut assignation
```suggestion
    end_date: transformDate,
```

Reply:
c0f98b4